### PR TITLE
chore: update pt-BR books

### PR DESCRIPTION
**Pull request by souzagab**
## What does this PR do?

Adds book references in pt-BR:
- Docker 
- Kubenetes

### Why is this valuable (or not)?
Good references in docker/kubernetes

### How do we know it's really free?
Open Source by [@badtuxx](https://github.com/badtuxx)
### For book lists, is it a book? For course lists, is it a course? etc.
 Books

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

### CODE REVIEW

File: books/free-programming-books-pt_BR.md [hunk 1]
```diff
@@ -275,6 +276,7 @@
 
 ### Kubernetes
 
+* [Descomplicando Kubernetes](https://github.com/badtuxx/DescomplicandoKubernetes/blob/main/SUMMARY.md) - Jeferson Fernando
```
Comment (davorpa) [inline books/free-programming-books-pt_BR.md:279]: What about if url is replaced by

https://livro.descomplicandokubernetes.com.br

It's a bit simple same as docker link ;)

Comment (souzagab) [reply books/free-programming-books-pt_BR.md:279]: Oh, thanks for the tip, i'll change it 🥇

**Review by davorpa — APPROVED** [review]

File: books/free-programming-books-pt_BR.md (hunk 1 above)
Comment (souzagab) [reply books/free-programming-books-pt_BR.md]: > Oh, thanks for the tip, i'll change it 🥇

https://github.com/EbookFoundation/free-programming-books/pull/5527/commits/a353c5e389223bf24a4a51d8b5b6a088abad8e27

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**